example/90-shoot.yaml

```
@@ -45,6 +45,12 @@ spec:
         # providerConfig:
         #   <some-machine-image-specific-configuration>
       # architecture: <some-cpu-architecture>
+    # autoscaler:
```

Maybe 
```suggestion
    # clusterAutoscaler:
```

to align with the fields in `.spec.kubernetes`?